include/deal.II/grid/tria_objects.h

```diff
@@ -301,14 +301,6 @@ namespace internal
       void serialize(Archive &ar,
                      const unsigned int version);
 
-      /**
-       * Exception
-       */
-      DeclException3 (ExcMemoryWasted,
-                      char *, int, int,
-                      << "The container " << arg1 << " contains "
-                      << arg2 << " elements, but it`s capacity is "
-                      << arg3 << ".");
```

Comment:
I noticed this a few months ago, remarked that it was never used, but forgot to follow up. Such banal changes are necessary to keep the code clean :+1: